sql/core/src/main/scala/org/apache/spark/sql/sources/interfaces.scala

```diff
@@ -308,8 +303,6 @@ trait InsertableRelation {
  *
  * @since 1.3.0
  */
-@Experimental
-@Unstable
```

Comment:
This should remain unstable, as it uses private classes `Attribute` `Expression`.